nautobot/extras/api/nested_serializers.py

```diff
@@ -138,3 +139,25 @@ class NestedRelationshipAssociationSerializer(WritableNestedSerializer):
     class Meta:
         model = models.RelationshipAssociation
         fields = ["id", "url", "relationship", "source_id", "destination_id"]
+
+
+class NestedJobScheduleSerializer(serializers.Serializer):
```

Comment:
Hm. Looking at the rest of this file as context, why isn't this a `WritableNestedSerializer` subclass? Why aren't we specifying `Meta.model` and `Meta.fields`?

Reply:
Fixed!